packages/gestalt/src/Checkbox/InternalCheckbox.tsx

```diff
@@ -180,6 +182,12 @@ const InternalCheckboxWithForwardRef = forwardRef<HTMLInputElement, Props>(funct
     [styles.readOnly]: readOnly,
   });
 
+  const dataTestIdInput = dataTestId && `${dataTestId}-input`;
```

Comment:
I think we can set the rule/pattern that html tags get the data-test-id with no append? That's consistent with Button, TextField etc.  

I'd do <input ` data-test-id={dataTestId` >

Thoughts?

Reply:
Makes sense to me, working on a commit for this update, thanks for catching this one

Reply:
Change ready